src/api/videoRoom.ts

```diff
@@ -0,0 +1,79 @@
+import Bugsnag from "@bugsnag/js";
+import firebase from "firebase/app";
+
+import { VideoChatRequest, VideoChatRequestState } from "types/VideoRoom";
+
+export const inviteToVideoChat = async (
```

Comment:
To align with everything else?

```suggestion
export const inviteToVideoRoom = async (
```